pact.cabal

```diff
@@ -136,6 +136,7 @@ library
                      , containers >= 0.5.7 && < 0.6
                      , data-default >= 0.7.1.1 && < 0.8
                      , deepseq >= 1.4.2.0 && < 1.5
+                     , deriving-compat >= 0.5.1
```

Comment:
Assume this is for `Eq1` and `Show1` instances, are we comfortable using them?

Reply:
What's the objection to them?

Reply:
Just being terrified that they won't work someday, but that nightmare has happened multiple times already so oh well